int32 num_yaw
float32 fov_yaw
float32 delta_yaw
